src/main/java/com/google/devtools/build/lib/sandbox/DockerSandboxedSpawnRunner.java

```diff
@@ -236,8 +236,7 @@ protected SpawnResult actuallyExec(Spawn spawn, SpawnExecutionContext context)
         .setEnvironmentVariables(environment)
         .setKillDelay(timeoutKillDelay)
         .setCreateNetworkNamespace(!(allowNetwork || Spawns.requiresNetwork(spawn)))
-        .setCommandId(commandId)
-        .setUuid(uuid);
```

Comment:
Why do you delete this line? If we don't set the UUID of the container, we can't clean it up later here:

https://source.bazel.build/bazel/+/master:src/main/java/com/google/devtools/build/lib/sandbox/DockerSandboxedSpawnRunner.java;l=391?q=DockerSandbox

I'm really not proud of this mechanism (generating a UUID for each container, keeping it in a list and then later deleting all containers that have a UUID from the list) - if you find anything better (there probably is!), I'd love to hear about it and accept a patch :)

Reply:
This was a mistake. Now fixed.